docs/_posts/2018-09-20-user-manual-table-of-contents.md

```diff
@@ -96,11 +96,9 @@ weekly to [GitHub](https://github.com/miso-lims/miso-lims/releases).
 {% assign sub = sub | plus: 1 %}
 {% include userman-toc-link.md section=section sub=sub section-title=section-title title="Tools" %}
 {% assign sub = sub | plus: 1 %}
-{% include userman-toc-link.md section=section sub=sub section-title=section-title title="My Account Tab" %}
+{% include userman-toc-link.md section=section sub=sub section-title=section-title title="My Account" %}
 {% assign sub = sub | plus: 1 %}
-{% include userman-toc-link.md section=section sub=sub section-title=section-title title="My Projects Tab" %}
-{% assign sub = sub | plus: 1 %}
-{% include userman-toc-link.md section=section sub=sub section-title=section-title title="Help Tab" %}
+{% include userman-toc-link.md section=section sub=sub section-title=section-title title="Help" %}
```

Comment:
Are we losing a section for the Projects page?

Reply:
No. This is the subsection of General Navigation that talks about the "My Projects" tab. Since that is now a link in the navigation menu, it's covered by the List Pages subsection of General Navigation, and of course by the entire Projects section